node_helper.js

```diff
@@ -19,37 +19,31 @@ module.exports = NodeHelper.create({
         self.url = this.quoteConfig.url;
         self.language = this.quoteConfig.language;
 
-        let options = {
-            url: self.url,
-            json: true,
-            method: "GET"
-        };
-
-        request(options, function (error, response, body) {
-            if (error) {
-                return console.log(error);
-            }
-            let strObject = JSON.stringify(body, null, 4);
-            console.log(strObject);
-            self.returned_data = body;
-
-            if (self.language !== "en") {
-                translate(self.returned_data.quoteText, {
-                    to: "fr"
-                }).then(res => {
-                    // console.log(res.text);
-                    self.returned_data.quoteText = res.text;
-                    self.sendSocketNotification('QUOTE_RESULT', self.returned_data);
-
-                }).catch(err => {
-                    console.error(err);
+        axios.get(self.url)
+            .then(function (response) {
+                console.log(response);
+                self.returned_data = response.data;
+
+                if (self.language !== "en") {
+                    translate(self.returned_data.quoteText, {
+                        to: "fr"
```

Comment:
just noticed that it should be `self.language` here.

Reply:
Addressed the comment. Thank you.